docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - POSTGRES_PORT=5432
       - WEB_CONCURRENCY=10
       - DB_MIN_CONN_SIZE=1
-      - DB_MAX_CONN_SIZE=1
+      - DB_MAX_CONN_SIZE=10
```

Comment:
when having only 1 conn I get `psycopg2.pool.PoolError: connection pool exhausted`

Reply:
I think you need to have DB_MAX_CONN_SIZE at least the same size as WEB_CONCURRENCY